packages/react/src/header/HeaderBrand.tsx

```diff
@@ -29,7 +29,7 @@ type HeaderBrandProps = Pick<
 
 export function HeaderBrand({
 	badgeLabel,
-	dividerPosition,
+	dividerPosition = 'after',
```

Comment:
Is this necessary since Header has this default?

Reply:
Ahh good point, it is not!

- [ ] Remove default `dividerPosition`